convert2rhel/unit_tests/redhatrelease_test.py

```diff
@@ -88,6 +97,50 @@ def test_patch_yum_conf_existing_distroverpkg(self):
             self.assertEqual(yum_conf._yum_conf_content.count("#distroverpkg="), 1)
 
 
+@pytest.mark.parametrize(
+    ("pkg_type, subprocess_ret, expected_result"),
+    (
+        ("dnf", "S.5....T.  c /etc/dnf/dnf.conf", True),
+        ("dnf", ".......T.  c /etc/dnf/dnf.conf", False),
+        ("dnf", ".M.......  g /var/lib/dnf", False),
+        ("yum", "S.5....T.  c /etc/yum.conf", True),
+        ("yum", "", False),
+        ("yum", ".......T.  c /etc/yum.conf", False),
+        ("unknown", "anything", False),
+    ),
+)
+def test_is_modified(monkeypatch, pkg_type, subprocess_ret, expected_result):
+    monkeypatch.setattr(pkgmanager, "TYPE", value=pkg_type)
+
+    run_subprocess = mock.Mock(return_value=(subprocess_ret, "anything"))
+    monkeypatch.setattr(utils, "run_subprocess", value=run_subprocess)
+
+    assert YumConf.is_modified() == expected_result
+
+
+@pytest.mark.parametrize("modified", [True, False])
```

Comment:
Nit: if you just need an iterable, without the need to modify it, then always use tuple (True, False) rather than a mutable list. That is especially important when you provide a default value in the function argument (read this https://web.archive.org/web/20200221224620/http://effbot.org/zone/default-values.htm)